crates/sui-sdk/src/lib.rs

```diff
@@ -76,10 +77,10 @@ impl SuiClientBuilder {
         let client_version = env!("CARGO_PKG_VERSION");
         let mut headers = HeaderMap::new();
         headers.insert(
-            "client_api_version",
```

Comment:
@patrickkuo fyi this is changed from `client_api_version` to `client-api-version`. Let me know if there's any concern

Reply:
On second thought, `client-target-api-version` might be a better name because the version number of TS SDK(and all community SDKs) is decoupled with RPC API, so we should use the targeted RPC API version instead

Reply:
ya lets change this to client-target-api-version